be/src/cloud/pb_convert.cpp

```diff
@@ -185,6 +221,9 @@ void cloud_rowset_meta_to_doris(RowsetMetaPB* out, const RowsetMetaCloudPB& in)
     if (in.has_tablet_schema()) {
         cloud_tablet_schema_to_doris(out->mutable_tablet_schema(), in.tablet_schema());
     }
+    if (dict != nullptr) {
```

Comment:
If dict is nullptr due to some exception and schema is not filled by meta service, the state is not valid. So you need to check the state if dict is nullptr.

Reply:
if exception happend, then the response will fail, and retry.